Src/BootCamp.Chapter/TextBoxPrinter.cs

```diff
@@ -1,12 +1,137 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
     public static class TextBoxPrinter
     {
-        // TODO: Implement.
-        public static string Print(object obj) => "";
+        public static string Print(object obj)
+        {
+            var atri = AttributesGetter.GetClassAttribute<TextableAttribute>(obj.GetType());
+            if (atri != null)
+            {
+                return MakePadding(obj.ToString(), atri.Padding, atri.Horizontal, atri.Vertical, atri.Corner);
+            }
+            else
+            {
+                return obj.ToString();
+            }
+        }
+
+        public static string MakePadding(string message, int padding = 0, char horizontal = '-', char vertical = '|', char corner = '+')
+        {
+            string[] splitmessage = message.Split(Environment.NewLine);
+            int height = splitmessage.Length + (padding * 2) + 1;
+            int width = 0;
+
+            foreach (string line in splitmessage) // checking which word is the longest to know how wide the padding should be.
+            {
+                if (line.Length > width)
+                {
+                    width = line.Length;
+                }
+            }
+            width += (padding * 2) + 1;
+
+            return MakeTabel();
+
+            string MakeTabel()
```

Comment:
Have a nested method like this beats the purpose: then purpose of nested methods is either reusability or clarification:
1) You do not reuse anything
2) the method is still inside the function, so you don't really hide a step

What you should be doing instead, is- use a private method in a class scope and move it below all the public methods, so that the method becomes an optional detail that you could refer to when reading the method, but are not enforced to.